ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -52,7 +50,7 @@ so no instance of this class ever gets created.  It starts up the application, a
   private static Texture clipboardTexture[];
   private static Material clipboardMaterial[];
   private static ImageMap clipboardImage[];
-  private static ArrayList<EditingWindow> windows = new ArrayList<EditingWindow>();
+  private static LinkedList<EditingWindow> windows = new LinkedList<EditingWindow>();
```

Comment:
Changing list implementations should have a specific reason. What makes you prefer a `LinkedList` here?

Reply:
This was tied to next changes in code where LinkedList's removeLast() was uses. Seems no more actual.

Reply:
That works for that one.